### PR TITLE
Update firstboot2.sh

### PR DESCRIPTION
**Pull request by npaladin2000**
Add automatic creation of ROM folders defined in es_systems.cfg and assign correct ownership and permissions (odroid:odroid, 755)

### CODE REVIEW

**Comment by valadaa48** [comment]
just one small change to be 100% safe (in case any dirs are there) - make them all mkdir -p

File: system-odroidgo2/firstboot2.sh [hunk 1]
```diff
@@ -6,5 +6,61 @@ mkdir -p /var/lib/alsa
...
+mkdir -p /roms/sg-1000/
+mkdir -p /roms/mastersystem/
+mkdir -p /roms/megadrive/
+mkdir -p /roms/genesis/
+mkdir -p /roms/segacd/
+mkdir -p /roms/sega32x/
+mkdir -p /roms/gamegear/
+mkdir -p /roms/ngp/
+mkdir -p /roms/ngpc/
+mkdir -p /roms/tg16cd</path>
```
Comment (valadaa48) [inline system-odroidgo2/firstboot2.sh:45]: I don't think this was intended?

File: system-odroidgo2/firstboot2.sh [hunk 2]
```diff
@@ -6,5 +6,61 @@ mkdir -p /var/lib/alsa
...
+mkdir -p /roms/tg16</path>
```
Comment (valadaa48) [inline system-odroidgo2/firstboot2.sh:46]: here

File: system-odroidgo2/firstboot2.sh (hunk 1 above)
Comment (npaladin2000) [reply system-odroidgo2/firstboot2.sh]: Oops.

**Comment by npaladin2000** [comment]
Ok I think I fixed all my typos now.

**Pull request merged**